cheatsheet.rst

```diff
@@ -418,9 +418,9 @@ A package recipe is a Python class, defined in a file called conanfile.py:
             ...                                                          # tasks, and using them is often preferable to using self.run()
                                                                          # See the link below for more information
 
-        def build_requirements(self):                                    # Responsible for specifying non-trivial build requirements logic
-            if self.options.myoption1:                                   # Specify a conditional build requirement
-                self.build_requires("zlib/1.2@user/testing")
+        def tool_requirements(self):                                    # Responsible for specifying non-trivial build requirements logic
```

Comment:
Does the `tool_requirements` method exist?

Reply:
No it doesn't